lib/fuel/configuration.rb

```diff
@@ -22,6 +21,8 @@ def initialize
       @paginates_per = 5
       @facebook_app_id = nil
       @site_name = nil
+      @authors = ['John Doe', 'Jane Smith']
+      @tags = ['Tag 1', 'Tag 2']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.